docker/mariadb/00_database-create-geokrety-db.sql

```diff
@@ -400,59 +308,60 @@ CREATE TABLE `gk-races-krety` (
   `finishCaches` bigint(20) NOT NULL COMMENT 'caches na chwilę zakończenia rajdu',
   `finishLat` double DEFAULT NULL,
-  `finishLon` double DEFAULT NULL
+  `finishLon` double DEFAULT NULL,
+  UNIQUE KEY `raceGkId` (`raceGkId`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='uczestnicy rajdów';
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-ruchy`
---
 
 CREATE TABLE `gk-ruchy` (
-  `ruch_id` int(10) UNSIGNED NOT NULL,
-  `id` int(10) UNSIGNED NOT NULL,
-  `lat` double(8,5) DEFAULT NULL COMMENT 'latitude',
-  `lon` double(8,5) DEFAULT NULL COMMENT 'longitude',
-  `alt` int(5) NOT NULL DEFAULT '-32768' COMMENT 'altitude',
-  `country` varchar(3) COLLATE utf8_polish_ci NOT NULL DEFAULT '' COMMENT 'country code',
-  `droga` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'road traveled in km',
-  `waypoint` varchar(10) COLLATE utf8_polish_ci NOT NULL DEFAULT '' COMMENT 'waypoint code - empty for comment',
-  `data` datetime DEFAULT NULL COMMENT 'ruchy user provided date',
-  `data_dodania` datetime DEFAULT NULL COMMENT 'ruchy database added date',
-  `user` int(10) UNSIGNED DEFAULT '0' COMMENT 'ruchy author user id',
-  `koment` varchar(5120) COLLATE utf8_polish_ci DEFAULT NULL COMMENT 'ruchy user provided comment',
-  `zdjecia` tinyint(3) UNSIGNED NOT NULL DEFAULT '0',
-  `komentarze` smallint(5) UNSIGNED NOT NULL DEFAULT '0',
+  `ruch_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `id` int(10) unsigned NOT NULL,
+  `lat` double(8,5) DEFAULT NULL,
+  `lon` double(8,5) DEFAULT NULL,
+  `alt` int(5) NOT NULL DEFAULT '-32768',
+  `country` varchar(3) COLLATE utf8_polish_ci NOT NULL,
+  `droga` int(10) unsigned NOT NULL,
+  `waypoint` varchar(10) COLLATE utf8_polish_ci NOT NULL,
+  `data` datetime DEFAULT NULL,
+  `data_dodania` datetime DEFAULT NULL,
+  `user` int(10) unsigned DEFAULT '0',
+  `koment` varchar(5120) COLLATE utf8_polish_ci DEFAULT NULL,
+  `zdjecia` tinyint(3) unsigned NOT NULL DEFAULT '0',
+  `komentarze` smallint(5) unsigned NOT NULL DEFAULT '0',
   `logtype` enum('0','1','2','3','4','5','6') COLLATE utf8_polish_ci DEFAULT '0' COMMENT '0=drop, 1=grab, 2=comment, 3=met, 4=arch, 5=dip',
   `username` varchar(20) COLLATE utf8_polish_ci NOT NULL,
   `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   `app` varchar(16) COLLATE utf8_polish_ci NOT NULL DEFAULT 'www' COMMENT 'source of the log',
-  `app_ver` varchar(16) COLLATE utf8_polish_ci NOT NULL COMMENT 'apploction version/codename'
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='GK ruchy - geokrety movements';
-
--- --------------------------------------------------------
+  `app_ver` varchar(16) COLLATE utf8_polish_ci NOT NULL COMMENT 'apploction version/codename',
+  PRIMARY KEY (`ruch_id`),
+  KEY `id_2` (`id`),
+  KEY `waypoint` (`waypoint`),
+  KEY `user` (`user`),
+  KEY `lat` (`lat`),
+  KEY `lon` (`lon`),
```

Comment:
> I don't know the typical queries against this table, but as per amount of indexes looks like introducing some composite indexes may be helpful
> 
> Sounds like you're talking about gk-ruchy table, no?

Correct, this one

Reply:
In the `new-theme` branch, found sql queries on this table should be round up there https://github.com/geokrety/geokrety-website/blob/feature/new-theme/website/lib/geokrety/repository/TripRepository.class.php

Feel free to propose new indexes, I'll push them to new-theme (and/or staging branch) so we can test them.

FYI, this is the biggest table in the database, it actually contains 1713876 entries. So proper indexing would be nice. Database updates may take a certain amount of time ;)